src/App.tsx

```diff
@@ -1,7 +1,40 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type Props = {
+};
+
+type State = {
+  pressedKey: string,
+};
+
+export class App extends Component<Props, State> {
+  state = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.pressAnyKey);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.pressAnyKey);
+  }
+
+  pressAnyKey = (event: KeyboardEvent) => {
+    this.setState({
+      pressedKey: event.key,
```

Comment:
try using `event.code`, to check if the key is `space`, now you have this:
<img width="211" alt="image" src="https://user-images.githubusercontent.com/55802630/182557457-605abfec-d25f-4a98-9243-dffa64969711.png">

Reply:
Fixed:)